lms/djangoapps/course_api/blocks/views.py

```diff
@@ -85,6 +86,11 @@ class BlocksView(DeveloperErrorViewMixin, ListAPIView):
 
           Example: return_type=dict
 
+        * block_types: (list) Requested types of blocks. Possible values include sequential,
```

Comment:
Can you expand this docstring to indicate that this parameter is actually used to "filter" the response?